CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+## Contributions
```

Comment:
Should we also add some more info about contributing code?

Reply:
Added